app/controllers/WhiteListAuthFilter.scala

```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import com.gu.pandomainauth.action.UserRequest
+import config.Config
+import io.circe.Json
+import play.api.mvc._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class WhiteListAuthFilter(config: Config)(implicit ec: ExecutionContext) extends ActionFilter[UserRequest] {
```

Comment:
This entire class will go away after we add the permissions client to this project.

Reply:
https://github.com/guardian/workflow-frontend/pull/245